spec/how_is_spec.rb

```diff
@@ -109,14 +109,14 @@
   end
 
   # Disable 'cop' that is violated by every .generate_frontmatter() calls.
-  # rubocop:disable Style/BracesAroundHashParameters
+  # rubocop:disable Layout/BracesAroundHashParameters
   context '#generate_frontmatter' do
     it 'works with frontmatter parameter using String keys, report_data using String keys' do
       actual = nil
       expected = nil
 
       VCR.use_cassette("how-is-example-repository") do
-        actual = HowIs.generate_frontmatter({'foo' => "bar %{baz}"}, {'baz' => "asdf"})
+        actual = HowIs.generate_frontmatter({'foo' => "bar %<baz>s"}, {'baz' => "asdf"})
```

Comment:
what's the reasoning for this? I personally prefer `%{baz}`. I'd rather revert this and disable/modify the RuboCop rule accordingly, unless there's a strong reason to do otherwise.